tests/nextcloud_mysql.yml

```diff
@@ -25,3 +25,5 @@
     nextcloud_apps:
       files_external: ""  # enable files_external which is already installed in nextcloud
       calendar: "https://github.com/nextcloud/calendar/releases/download/v1.5.0/calendar.tar.gz"  # download and install calendar app
+    nextcloud_disable_apps:
```

Comment:
I think these tests are not used any more. These are some obsolete Vagrant files.

Reply:
@staticdev 
Yes they're not , but they're not for vagrant. I wrote them when I started the role and was using circleCI.
They seems to be good valid playbooks that could help with code coverage.

PS: I'm thinking about a way to do better code  coverage in our current molecule that would not take ages to do.
I'll have to do some tests with the `import_role` module and the `tasks_from` parameter for ad-hoc unit tests of each tasks file.
or maybe I'm reinventing the wheel and a project for doing that exists already.

Reply:
So maybe it's worth adding them to molecule. Obviously not in this PR, but maybe as a next step to test more use cases.